### ISSUE TITLE
Emacs sounds not working

### ISSUE THREAD

**Issue by steve-solomon** [microsoft/wslg#763]
### Summary
Emacs sounds (e.g. bell) not working

### Environment
Windows 11
Build 22000.675
WSL version:
```
Major  Minor  Build  Revision
-----  -----  -----  --------
10     0      22000  653
```

### Description
I installed Emacs 28 via `sudo apt install emacs`. When I enter keystrokes that should result in an audible bell, e.g. ctrl-g in the Emacs splash page, no sounds are made. Another: scrolling past the end of the current buffer using ctrl-v.

I also have Chrome installed, and sound *is* working, via the Pulse Audio plugins.

Tried setting `(visible-bell nil)` to no effect, while `(visible-bell t)` results in a visible flash as expected.

### Expected behavior
Audible bell sounds are made.

**Comment by hideyukn88** [comment]
@steve-solomon, thanks for reporting the issue. To address this issue, pluseaudio in our system distro needs to load "module-x11-bell" module with appropriate wav file uploaded. We will look into the fix, thanks!

**Comment by hideyukn88** [comment]
Fixed in WSLg 1.0.41
